cf-custom-resources/lib/alb-rule-priority-generator.js

```diff
@@ -150,7 +150,7 @@ exports.nextAvailableRulePriorityHandler = async function (event, context) {
       "FAILED",
       physicalResourceId,
       null,
-      err.message
+      `${err.message} (Log: ${context.logStreamName}/${context.logGroupName})`
```

Comment:
Just throwing this out: what about `Reason: ${reason} (Log: ${context.logStreamName}/${context.logGroupName})` in `report`? This is great too though so no need to change it.